cloudinit/config/cc_set_passwords.py

```diff
@@ -111,6 +111,15 @@ def handle_ssh_pwauth(pw_auth, distro: Distro):
                 service,
             )
             return
+        elif not uses_systemd:
```

Comment:
It probably makes sense to shuffle this condition up first so we can drop the `if uses_systemd and ...` in the other conditionals
```python
if not distro.uses_systemd():
    <your new debug log>
elif e.exit_code == 3:
   <existing exit_code 3 behavior>
elif e.exid_code == 4:
   < existing exit_code 4 behavior>
else:
   LOG.warning(Ignoring...)
   return
```

Reply:
Simplified. Thanks for the suggestion @blackboxsw.